contracts/libraries/Constants.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.15;
+
+/// @title Constant state
+/// @notice Constant state used by the swap router
+library Constants {
+    /// @dev Used for identifying cases when this contract's balance of a token is to be used
+    uint256 internal constant CONTRACT_BALANCE = 0;
```

Comment:
What if we used the following? ...
`uint256 internal constant CONTRACT_BALANCE = uint256(-69);`

Reply:
Does it matter though? It only transfers when not 0 and if balance is actually 0 then it will most likely fail somewhere else where it needs tokens. Or it's expected behavior for 0 tokens probably